content/docs/gto/command-reference/register.md

```diff
@@ -0,0 +1,63 @@
+# register
+
+Create an artifact version to signify an important, published or released
+iteration.
```

Comment:
Maybe we should be more specific on what it does, like:
```suggestion
Create a Git tag containing the artifact's name and version.
```

> Not sure, e.g. in DVC sometimes we keep it general (https://dvc.org/doc/command-reference/commit) and sometimes specific (https://dvc.org/doc/command-reference/checkout).

I guess it depends on whether we expect users to be familiar enough with Git and/or whether we want them to keep in mind the mechanics. But if we consider these implementation details, then let's keep it general but also remove/ hide Git tag details in general.

Reply:
Since the pages are auto-generated, this requires changing the code. Will do that later.